test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -209,7 +209,16 @@ class test_ucp_sockaddr : public ucp_test {
                                           ucp_listener_h &listener)
     {
         scoped_log_handler wrap_err(wrap_errors_logger);
-        return ucp_listener_create(receiver().worker(), &params, &listener);
+
+        /* FIXME: in spite of using SO_REUSEPORT tcp CM can return addr in use error
+         *        for specific port acquired by rdmacm CM */
```

Comment:
does it mean tcp can't use same port as rdmacm?
or it means that it happens after rdma_cm already destroyed its id?
given this, do we still need to set SO_REUSEPORT?

Reply:
TCP can share the port with RDMACM, it means that some port can be free in RDMACM but busy in TCP port space. When RDMACM aquires some port from ANY we shoud start TCP on the same port but it can be busy, for example, by other app